ArchiSteamFarm/Bot.cs

```diff
@@ -685,7 +685,27 @@ internal sealed class Bot : IDisposable {
 			return await ArchiWebHandler.GetTradeHoldDurationForTrade(tradeID).ConfigureAwait(false);
 		}
 
-		internal async Task<(Dictionary<string, string> Used, Dictionary<string, string> Unused)> GetUsedAndUnusedKeys() => (await GetKeysFromFile(KeysToRedeemUsedFilePath).ConfigureAwait(false), await GetKeysFromFile(KeysToRedeemUnusedFilePath).ConfigureAwait(false));
+		internal async Task<(Dictionary<string, string> Used, Dictionary<string, string> Unused)> GetUsedAndUnusedKeys() {
+			ICollection<string> fileNames = new List<string>(new string[] { KeysToRedeemUsedFilePath, KeysToRedeemUnusedFilePath });
```

Comment:
This hurts my eyes, just `Task<bool> task1 = GetKeysFromFile(file1)` and `Task<bool> task2 = GetKeysFromFile(file2)`, then you can insert 2x `await` in min memory patern and `Task.WhenAll(task1, task2)` in second one.

Reply:
Actually even that 2x `await` is useless, you can just do `new List<string> { await task1, await task2 }`.